### ISSUE TITLE
Reflections poorly supported/documented

### ISSUE THREAD

**Issue by rivertam** [rustsim/nalgebra#639]
TL;DR: I'd like to be able to reflect an `Isometry3` about a plane, including both translation and rotation components. I see no reason this would be impossible, but I can't figure out how to do so given the documentation.

---

Hi! I'm trying to use nalgebra for some relatively simple 3D geometry utilities, and one thing I'd like to be able to do is go from my physical world (Z is up and XY is the ground plane) to the coordinate system CSS uses (which I think is pretty common): Y is down, X is left-to-right, and Z goes inward (so it's left-handed, I'm pretty sure).

nalgebra is so far the geometry library I know best for any language, so there might be a better way of doing this, but I don't know what to look for. At the moment, the way I'm trying to do this is:

1. Rotate isometry about the x axis (pi / 2)
2. Negating the y component of the translation
3. Rotating just the rotation about the Z axis

I did this out on paper, and I think it should work, but I can't get the implementation right. I'm definitely not asking you to do my homework (not that this is homework), but I feel this should be much simpler than it is.
